src/Domain.LinnApps/Stock/StoresPallet.cs

```diff
@@ -4,18 +4,146 @@
 
     public class StoresPallet
     {
-        public int PalletNumber { get; set; }
+        public StoresPallet()
+        {
+        }
+
+        public StoresPallet(
+            int palletNumber,
+            string description,
+            StorageLocation locationId,
+            DateTime? dateInvalid,
```

Comment:
probably don't need to initialise with a dateinvalid or datelastaudited or any field that only really gets updated